kalitectl.py

```diff
@@ -592,6 +605,85 @@ def status():
 }
 
 
+def diagnose():
+    """
```

Comment:
Ooh, any way we could default to `traceback = True` for this command, so that in the case even this command fails, the thing they copy will provide useful feedback to us?